package.json

```diff
@@ -42,88 +42,88 @@
-    "cronstrue": "1.113.0",
-    "d3": "5.16.0",
-    "d3-transition": "1.2.0",
-    "date-fns": "2.10.0",
+    "angular-calendar": "~0.28.2",
+    "angular-draggable-droppable": "~4.3.8",
+    "angular-resizable-element": "~3.3.0",
+    "angular-svg-icon": "~9.0.0",
+    "angular-tree-component": "~8.5.6",
+    "angular2-qrcode": "~2.0.3",
+    "angular2-uuid": "~1.1.1",
+    "chart.js": "~2.9.4",
+    "chartist": "~0.11.4",
+    "classlist.js": "~1.1.20150312",
+    "core-js": "~3.6.4",
+    "cron-parser": "~2.13.0",
+    "cronstrue": "~1.113.0",
+    "d3": "~5.16.0",
+    "d3-transition": "~1.2.0",
+    "date-fns": "~2.10.0",
     "date-fns-tz": "^1.1.4",
-    "dygraphs": "2.1.0",
-    "element-resize-detector": "1.2.1",
+    "dygraphs": "~2.1.0",
+    "element-resize-detector": "~1.2.1",
     "eslint-plugin-rxjs": "^3.3.3",
     "eslint-plugin-rxjs-angular": "^1.0.6",
     "eslint-plugin-unicorn": "^34.0.1",
-    "filesize": "6.3.0",
+    "filesize": "~6.3.0",
     "fontfaceobserver": "^2.1.0",
-    "hammerjs": "2.0.8",
-    "hopscotch": "0.3.1",
-    "ip-regex": "4.1.0",
-    "is-cidr": "3.1.0",
+    "hammerjs": "~2.0.8",
+    "hopscotch": "~0.3.1",
+    "ip-regex": "~4.1.0",
+    "is-cidr": "~3.1.0",
     "jest-canvas-mock": "^2.3.1",
     "jest-preset-angular": "^9.0.3",
     "jest-when": "^3.3.1",
-    "lodash": "4.17.21",
-    "messageformat": "2.3.0",
-    "messageformat-parser": "4.1.3",
-    "moment": "2.24.0",
-    "moment-timezone": "0.5.33",
-    "ng-chartist": "4.1.0",
+    "lodash": "~4.17.21",
+    "messageformat": "~2.3.0",
+    "messageformat-parser": "~4.1.3",
+    "moment": "~2.24.0",
+    "moment-timezone": "~0.5.33",
+    "ng-chartist": "~4.1.0",
     "ng-mocks": "^12.4.0",
-    "ng2-dragula": "2.1.1",
-    "ngx-color-picker": "9.0.0",
-    "ngx-filesize": "2.0.2",
+    "ng2-dragula": "~2.1.1",
+    "ngx-color-picker": "~9.0.0",
+    "ngx-filesize": "~2.0.2",
     "ngx-img-fallback": "^1.3.4",
-    "ngx-markdown": "10.0.0",
-    "ngx-translate-messageformat-compiler": "4.9.0",
-    "ngx-uploader": "8.0.0",
-    "ngx-webstorage": "5.0.0",
-    "pixi-filters": "2.7.1",
-    "pixi.js": "4.7.1",
-    "popmotion": "8.7.1",
-    "primeicons": "2.0.0",
-    "primeng": "9.0.0",
-    "q": "1.5.1",
-    "rxjs": "6.6.0",
-    "stylefire": "7.0.2",
-    "what-is-my-ip-address": "1.0.3",
-    "xterm": "4.9.0",
-    "xterm-addon-fit": "0.4.0",
-    "zone.js": "0.10.3"
+    "ngx-markdown": "~10.0.0",
+    "ngx-translate-messageformat-compiler": "~4.9.0",
+    "ngx-uploader": "~8.0.0",
+    "ngx-webstorage": "~5.0.0",
+    "pixi-filters": "~2.7.1",
+    "pixi.js": "~4.7.1",
+    "popmotion": "~8.7.1",
+    "primeicons": "~2.0.0",
+    "primeng": "~9.0.0",
+    "q": "~1.5.1",
+    "rxjs": "~6.6.0",
+    "stylefire": "~7.0.2",
+    "what-is-my-ip-address": "~1.0.3",
+    "xterm": "~4.9.0",
+    "xterm-addon-fit": "~0.4.0",
+    "zone.js": "~0.10.3"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "0.1000.1",
```

Comment:
We can add ~ to devDependencies too.

Reply:
That causes the CI build image to fail for some reason